chaosblade-exec-plugin/chaosblade-exec-plugin-dubbo/src/main/java/com/alibaba/chaosblade/exec/plugin/dubbo/provider/DubboProviderEnhancer.java

```diff
@@ -41,8 +43,14 @@ protected TimeoutExecutor createTimeoutExecutor(ClassLoader classLoader, long ti
     }
 
     @Override
-    protected Map<String, Map<String, String>> getBusinessParams(Object invocation) throws Exception {
-        return null;
+    protected Map<String, Map<String, String>> getBusinessParams(final Object invocation) throws Exception {
+        return BusinessParamUtil.getAndParse(DubboConstant.TARGET_NAME, new BusinessDataGetter() {
+            @Override
+            public String get(String key) throws Exception {
+                Map<String, String> attachments = ReflectUtil.invokeMethod(invocation, GET_ATTACHMENTS, new Object[0], false);
+                return attachments.get(key);
```

Comment:
The `attachments` may be null, please judge the value first.

Reply:
done